ndb/src/google/cloud/ndb/model.py

```diff
@@ -1387,9 +1387,42 @@ def __init__(self, *args, **kwargs):
 
 
 class BooleanProperty(Property):
+    """A property that contains values of type bool."""
+
     __slots__ = ()
 
-    def __init__(self, *args, **kwargs):
+    def _validate(self, value):
+        """Validate a ``value`` before setting it.
+
+        Args:
+            value (bool): The value to check.
+
+        Returns:
+            bool: The passed-in ``value``.
+
+        Raises:
+            .BadValueError: If ``value`` is not a :class:`bool`.
+        """
```

Comment:
No lax values or coercion?

Reply:
Not in the original and IMO that is the "correct" implementation.

Reply:
Ok.